recipes/giotto-learn/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 4710ef554ab4f117fcf3953635a268526e7ea15f57f1e88655ffb425e567e646
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  skip: true  # [win and vs<17]
+  script:
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pybind11
+    - boost >=1.56.*
+    - setuptools
+    - pip
+    - numpy >=1.17.0
+    - scipy
+    - scikit-learn >=0.21.3
```

you also need ``openmp``
```suggestion
    - scikit-learn >=0.21.3
    - openmp
```